package.json

```diff
@@ -7,15 +7,16 @@
     "@doc-tools/transform": "^2.16.0",
     "@gravity-ui/page-constructor": "^1.7.0",
     "@gravity-ui/uikit": "^3.18.0",
+    "@mdx-js/runtime": "^1.6.22",
```

Comment:
Looks like abandoned library https://www.npmjs.com/package/@mdx-js/runtime

Last version published 2 years ago, it has no typings and it has warning "⚠️ this is not the preferred way to use MDX"

Reply:
Migrated from `@mdx-js/runtime` to `@mdx-js/mdx` and `@mdx-js/react`.